### DISCUSSION TITLE
Possible typo in supported board list

### DISCUSSION THREAD

**Discussion by alextrical** [stm32duino/Arduino_Core_STM32#1779] [General]
Hi All

Just been looking through the list of supported Chips, and it looks like there may be a typo, as I don't believe this chip exists, when comparing against the STM32 stock list? 
![image](https://user-images.githubusercontent.com/35117191/182578470-704d0adb-046d-4baf-a365-591463680303.png)

**Comment by ABOSTM** [comment]
Hi @alextrical,
You are right, it is a typo, thanks for highlighting it.
It should be STM32F217ZG
as well as STM32F207G should be replaced by STM32F207ZG.

Would you submit a PR to fix this ? (in Readme.txt)

**Reply by ABOSTM** [reply]
Fixed thanks to #1788
